### ISSUE TITLE
Switch default branch to `master` from `develop`

### ISSUE THREAD

**Issue by humphd** [filerjs/filer#383]
It's pretty rare for projects to use a non-`master` branch as their default these days, and I'd love to not surprise new contributors.  @modeswitch would be OK if I reset `master` to `develop` and changed the repo's default branch to `master`?  I think this will help my students as they get started.

**Comment by modeswitch** [comment]
Yeah, that's fine. I don't think we have a master branch at the moment.
